This patch replaces the dependency on nose with a dependency on nose3.

--- 1.6.0/foss-2023a/yaff/yaff-1.6.0/setup.py.orig	2023-12-11 18:18:12.215725329 +0000
+++ 1.6.0/foss-2023a/yaff/yaff-1.6.0/setup.py	2023-12-11 18:19:04.794458554 +0000
@@ -90,7 +90,7 @@
     include_package_data=True,
     zip_safe=False,
     setup_requires=['numpy>=1.5', 'cython>=0.26'],
-    install_requires=['numpy>=1.5', 'nose>=0.11', 'cython>=0.26', 'matplotlib>1.0.0',
+    install_requires=['numpy>=1.5', 'nose3', 'cython>=0.26', 'matplotlib>1.0.0',
                       'h5py>=2.0.0', 'molmod>=1.4.1', 'scipy>=0.17.1'],
     ext_modules=[
         Extension("yaff.pes.ext",
